scripts/download-json.ts

```diff
@@ -48,11 +48,11 @@ import fetch from 'node-fetch';
     console.log(`Progress: ${exercises.length}/${total_count}`);
   }
 
-  writeFile(filename, JSON.stringify(exercises), (err) => {
+  writeFile('exercises.json', JSON.stringify({title, exercises}), (err) => {
     if (err)
       console.error(err);
     else {
-      console.log(`${exercises.length} exercises written to ${filename}`);
+      console.log(`${exercises.length} exercises written to exercises.json`);
```

Comment:
At the moment we always want the data saving to the same file. I also added a title arg which is used as a heading for the pdf.